src/views/Login.js

```diff
@@ -71,26 +93,22 @@ export default function Login() {
     const _wallet = need.wallet(wallet);
     const _contracts = need.contracts(contracts);
 
-    // if no point cursor set by login logic, try to deduce it
+    setDeducing(true);
+
     let deduced = pointCursor;
+    // if no point cursor set by login logic, try to deduce it
     if (Nothing.hasInstance(deduced)) {
       const owned = await azimuth.azimuth.getOwnedPoints(
         _contracts,
         _wallet.address
       );
       if (owned.length === 1) {
         deduced = Just(owned[0]);
-      } else if (owned.length === 0) {
-        const canOwn = await azimuth.azimuth.getTransferringFor(
```

Comment:
removed because acceptance will be on the 'points' page, so we want those users to end up at points